doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -330,6 +330,12 @@ Bluetooth samples
 
       This change results from the Bluetooth subsystem transition to the PSA cryptographic API.
 
+    * The following samples to use LE secure connection pairing (:kconfig:option:`CONFIG_BT_SMP_SC_PAIR_ONLY`).
```

Comment:
```suggestion
    * The following samples to use Bluetooth LE secure connection pairing (:kconfig:option:`CONFIG_BT_SMP_SC_PAIR_ONLY`).
```

Reply:
The proper name is LE Secure Connection, it is used in the Bluetooth SIG documentation.